os-server/src/main/resources/templates/email/account-verification.html

```diff
@@ -0,0 +1,35 @@
+<!doctype html>
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Email</title>
+</head>
+<body>
+<div style="color: black; font-style: normal; font-size: 14px;">
+    Hey <b th:text="${userName}"></b>
+    <p>Click the button blow to verify your account.</p>
+    <p> It is valid until <b th:text="${date}"></b>.</p>
+
+    <form th:method="post"
+          th:action="@{http://localhost:5000/api/v1/auth/register/verification}"
```

Comment:
the hostname will be different for different environments. so please keep this in as property variable